wouso/resources/templates/cpanel/proposed_home.html

```diff
@@ -0,0 +1,237 @@
+{% extends 'cpanel/index.html' %}
+{% load i18n %}
+{% load user %}
+{% load django_bootstrap_breadcrumbs %}
+
+{% block sectiontitle %}
+    Proposed Question
+
+    {% block breadcrumbs %}
+
+        {{ block.super }}
+
+        {% breadcrumb "Proposed Questions" "proposed_home" %}
+
+    {% endblock %}
+
+{% endblock %}
+
+{% block sectioncontent %}
+
+    <div class="container">
+
+        <ul class="nav nav-tabs">
+            <li><a href="#workshop">Workshop</a></li>
+            <li><a href="#quest">Quest</a></li>
+            <li><a href="#special_challenge">Special Challenge</a></li>
+            <li class="active"><a href="#qotd">Question of the Day</a></li>
+            <li><a href="#challenge">Challenge</a></li>
+            <li><a href="#quiz">Quiz</a></li>
+        </ul>
+
+
+        <div class="tab-content">
+
+            <div id="workshop" class="tab-pane fade in active">
```

Comment:
@vladeugeniu isn't the class supposed to be fade-in? Right now you have `fade` and `in` as two separate classes